crates/nu-cli/src/completions/operator_completions.rs

```diff
@@ -0,0 +1,226 @@
+use crate::completions::{
+    Completer, CompletionOptions, MatchAlgorithm, SemanticSuggestion, SuggestionKind,
+};
+use nu_protocol::{
+    ast::*,
+    engine::{Stack, StateWorkingSet},
+    Span,
+};
+use reedline::Suggestion;
+
+#[derive(Clone)]
+pub struct OperatorCompletion {
+    previous_expr: Expression,
+}
+
+impl OperatorCompletion {
+    pub fn new(previous_expr: Expression) -> Self {
+        OperatorCompletion { previous_expr }
+    }
+}
+
+impl Completer for OperatorCompletion {
+    fn fetch(
+        &mut self,
+        working_set: &StateWorkingSet,
+        _stack: &Stack,
+        _prefix: Vec<u8>,
+        span: Span,
+        offset: usize,
+        _pos: usize,
+        _options: &CompletionOptions,
+    ) -> Vec<SemanticSuggestion> {
+        //Check if int, float, or string
+        let partial = std::str::from_utf8(working_set.get_span_contents(span)).unwrap_or("");
+        match &self.previous_expr.expr {
+            Expr::BinaryOp(x, _, _) => match x.expr {
+                Expr::Int(_) => fetch_int_completions(span, offset, partial),
+                Expr::String(_) => fetch_str_completions(span, offset, partial),
+                Expr::Float(_) => fetch_float_completions(span, offset, partial),
+                _ => vec![],
+            },
+            _ => vec![],
+        }
+    }
+}
+
+pub fn fetch_int_completions(span: Span, offset: usize, partial: &str) -> Vec<SemanticSuggestion> {
+    let int_ops = vec![
+        ("+", "Plus / Addition"),
+        ("-", "Minus / Subtraction"),
+        ("*", "Multiply"),
+        ("/", "Divide"),
+        ("=", "Equal"),
```

Comment:
Oh that is a serious mistake in [our docs](https://www.nushell.sh/lang-guide/chapters/operators.html#arithmetic-operators)!

```suggestion
        ("==", "Equal"),
```